### ISSUE TITLE
loss is getting low very quickly

### ISSUE THREAD

**Issue by hashirali2604** [thtrieu/darkflow#902]
I have a dataset of more than 16,000 images having 11 classes but now at a learning rate of 1e-03 my loss and ave loss is getting as low as power -8 in just 1 epoch..... and after evaluating the model it doesn't show any detection even on 0.01 threshold... what do i have to do?
also right now i started training on 1e-05 and now after 1200 iterations its ave loss is 0.0098.. and after 2 epochs its showing loss of power -5 and again upon detection nothing happen...
